### PR TITLE
added SPOJ

### PR DESCRIPTION
**Pull request by arin17bishwa**

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -5177,6 +5177,16 @@
         "known" : ["spicerex", "rod-it"],
         "cat" : "tech"
        },
+      {
+        "name" : "SPOJ",
+        "uri_check" : "https://www.spoj.com/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "<li>Profile</li>",
```
Comment (WebBreacher) [inline wmn-data.json:5184]: I'm not seeing this `e_string` in the response. Please consider changing it to something like `user-profile-tables`

File: wmn-data.json [hunk 2]
```diff
@@ -5177,6 +5177,16 @@
         "known" : ["spicerex", "rod-it"],
         "cat" : "tech"
        },
+      {
+        "name" : "SPOJ",
+        "uri_check" : "https://www.spoj.com/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "<li>Profile</li>",
+        "m_string" : " was not found on this server.",
```
Comment (WebBreacher) [inline wmn-data.json:5185]: Not seeing this `m_string`. Consider something like `<strong>Innopolis Open 2018</strong>`

File: wmn-data.json (hunk 1 above)
Comment (WebBreacher) [inline wmn-data.json:5184]: image

I'm not seeing the `e_string` in the response from the server for good/valid accounts. See the image where I visited the defrager user (your known user) and it responded with a valid user profile page. I then looked at the response and searched for the `e_string` and it was not found.

Comment (arin17bishwa) [reply wmn-data.json]: Hi, I must have searched for e_string while I was logged in, that would have produced the `Profile` element.
Corrected it.

**Pull request merged**